dataminer-overview/Support Services/User_Operations.Support.md

```diff
@@ -0,0 +1,81 @@
+---
+uid: User_operations_support
+---
+
+# User Operations Support application
+
+With the User Operations Support application, you can both keep track of existing tickets and create new ones.
+
+The application is available at <https://operations.skyline.be/>
+
+## Ticket Overview page
+
+The *Ticket Overview* page is the default page shown when you open the User Operations Support application. On this page, you can check existing tickets.
+
+Several **filters** are available on this page:
+
+- You can use the filters in the top-left corner to only show tickets with a specific **status**.
+
+- With the time filter at the top, you can filter the tickets on **creation time**.
+
+The check boxes right above the table allow you to show or hide specific columns. Below this, you can also select how many rows should be displayed at the same time.
+
+If you click a ticket ID in the table, the **ticket details** will be displayed. These include any **tasks** related to the ticket.
+
+## Report Ticket page
+
+The *Report Ticket* page allows you to report a new ticket:
+
+1. On the *Registration* tab, fill in the following information:
```

Comment:
Can we provide more information about what should be filled in in the various fields below? E.g. which fields are mandatory, where can users find information that might not be obvious for them, is any information filled in automatically, ...

Reply:
added info on the fields